src/CONST.js

```diff
@@ -394,6 +394,7 @@ const CONST = {
     EMOJI_PICKER_HEADER_HEIGHT: 38,
 
     COMPOSER_MAX_HEIGHT: 116,
+    CHAT_GHOST_ROW_HEIGHT: 80,
```

Comment:
Can we change all of the names everywhere to reference "skeleton" instead of "ghost"? When I think of "ghost" I think of an account that doesn't exist. I could be mistaken but the style of UI we're adding here is called a "skeleton screen".

Reply:
Done.